### PR TITLE
align some versions

### PR DESCRIPTION
**Pull request by freben**

### CODE REVIEW

File: packages/create-app/templates/default-app/yarn.lock [hunk 1]
```diff
@@ -0,0 +1 @@
+# intentionally left empty
```
Comment (freben) [inline packages/create-app/templates/default-app/yarn.lock:1]: without this, when running create-app locally, yarn3 yells that the newly created app seems to be an unexpected part of a larger repo (yeah it's inside the monorepo). Creating an empty yarn.lock file that gets immediately overwritten was the most simple fix

**Review by Rugvip — CHANGES_REQUESTED** [review]
✨

File: packages/create-app/templates/default-app/yarn.lock (hunk 1 above)
Comment (freben) [reply packages/create-app/templates/default-app/yarn.lock:1]: Talked about this offline and decided to keep for now

File: packages/create-app/templates/default-app/packages/app/package.json.hbs [hunk 2]
```diff
@@ -37,33 +48,22 @@
     "history": "^5.0.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-router": "6.0.0-beta.0",
-    "react-router-dom": "6.0.0-beta.0",
-    "react-use": "^15.3.3"
+    "react-router": "^6.3.0",
+    "react-router-dom": "^6.3.0",
```
[the remark this answers is not in the document]
Comment (freben) [reply packages/create-app/templates/default-app/packages/app/package.json.hbs:52]: Done!

File: packages/create-app/templates/default-app/packages/app/src/App.tsx [hunk 3]
```diff
@@ -81,10 +81,13 @@ const routes = (
       path="/tech-radar"
       element={<TechRadarPage width={1500} height={800} />}
     />
-    <PermissionedRoute
+    <Route
```
[the remark this answers is not in the document]
Comment (freben) [reply packages/create-app/templates/default-app/packages/app/src/App.tsx:84]: Done!

**Review by Rugvip — APPROVED** [review]
:shipit:

**Pull request merged**